client/components/shortcode/index.jsx

```diff
@@ -21,7 +21,7 @@ import * as ShortcodesActions from 'lib/shortcodes/actions';
 import ShortcodeData from './data';
 import ShortcodeFrame from './frame';
 
-export default class extends React.Component {
+export default class Shortcode extends React.Component {
 	static displayName = 'Shortcode';
```

Comment:
Do we need to keep the `displayName` property in this particular case?

Reply:
Thanks! Missed that one. Got it in 9b7bb905cef2f208e1ee23c08de97c3372c6627b